src/content/operations/nat-traversal/_index.en.md

```diff
@@ -356,9 +356,29 @@ NAT, or if other applications on the internal network were already using the 450
 ports, the remote ends would not be able to contact gateway C or D over the expected ports.
 {{% /notice %}}
 
-### Double NAT Traversal
+### Double NAT Traversal (scenario 1)
 
-In this case case, A & B cluster gateways have direct reachability over their private
+In this case, clusters C & D are neither reachable on their private IPs (192.168.0.4 and 192.168.0.4) nor
+on the public IP. However, they are reachable over the private floating IPs (10.2.0.1 and 10.2.0.2).
+Submariner cannot detect these private floating IPs. To get the connectivity working, you can annotate
+the Gateway node with the private floating IP as shown below.
+
+![on-premises](/images/natt/double_nat_scenario1.png)
+
+```bash
```

Comment:
Q : the annotated public IP (e.g: 10.2.0.1) will be used as EP's public IP for all connections (including to clusters C,D) ,  can you elaborate how the connection from cluster A to C will work ?

Reply:
There is no Cluster A in this topology. Please look at the next scenario.

Reply:
Ack, 10x